src/sonic-config-engine/minigraph.py

```diff
@@ -1115,7 +1115,23 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     results['ACL_TABLE'] = filter_acl_table_bindings(acls, neighbors, pcs, sub_role)
     results['FEATURE'] = {
         'telemetry': {
-            'status': 'enabled'
+            'status': 'enabled',
+            'set_owner': 'kube'
+        },
+        'snmp': {
+            'set_owner': 'kube'
```

Comment:
are we switch all of them to kube by default?

Reply:
Yes. This will be completely benign change, until switch joins and there is a manifest available for this feature,

Reply:
but if your build hasn't include_kubernates, then it should still be local.

Reply:
also, i feel it should not be, better to modify the init_cfg.json.j2 there to setup the default value.